build.sbt

```diff
@@ -290,12 +290,15 @@ lazy val ocs2 = project
 lazy val ephemeris = project
   .in(file("modules/ephemeris"))
   .enablePlugins(AutomateHeaderPlugin)
-  .dependsOn(coreJVM, db, sql)
+  .dependsOn(coreJVM % "compile->compile;test->test", db, sql)
```

Comment:
This is done to pick up the new test case tags, which are used to skip network tests.